tests/src/Simple/PInvoke/PInvoke.csproj

```diff
@@ -0,0 +1,11 @@
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <Compile Include="*.cs" />
+  </ItemGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), SimpleTest.targets))\SimpleTest.targets" />
+
+  <ItemGroup>
+     <ProjectReference Include="PInvokeNative.vcxproj"/>
```

Comment:
Why do you have this split into csproj and vcxproj? vcxproj is Windows MSVC++ specific. It won't work cross-plat. I think it should be all a single .csproj.